recipes/chart-studio/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "chart-studio" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1293776553a552bb3ff918c33d70f3b0794103b8cb0ac08aad8bde1c94ee65e3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - plotly
+    - requests
+    - retrying >=1.3.3
+    - six
+
+test:
+  imports:
+    - chart_studio
+
+about:
+  home: https://github.com/plotly/plotly.py
+  license: MIT
+  license_family: MIT
+  license_file: "/packages/python/chart-studio/LICENSE.txt"
```

Comment:
This file doesn't exist in the pypi tarball. Please add the License file from the repo (https://github.com/plotly/plotly.py/blob/master/LICENSE.txt), and change license_file to `license_file: LICENSE.txt`.